server/protocols/webadmin/webadmin-data/pom.xml

```diff
@@ -32,6 +32,23 @@
     <name>Apache James :: Server :: Web Admin :: data</name>
 
     <dependencies>
+        <dependency>
```

Comment:
If we move those tests into webadmin-mailbox then we do no longer need those dependencies here.